.editorconfig

```diff
@@ -0,0 +1,20 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
It's mostly for lowkey enforcement across editors - I mainly use it cos it doesn't the installation of any code dependencies to enforce its minimal rules. Most useful of those is the new line at the end of the file & indentation. Otherwise, it's harmless.

Reply:
Ohh that newline at the end of a file sounds nice! I may look into adding this into our template repos.